global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "6.0.100-preview.4.21216.8"
+    "version": "6.0.100-preview.4.21222.6"
```

Comment:
Heads up @dotnet/aspnet-build. I'm updating the preview4 branch SDK version so we can consume the `WebApplication` related types from our new "empty" web project templtates.

Reply:
If these new types come from dotnet/aspnetcore, something is very wrong if they aren't just showing up. Fine if, on the other hand, the main thing we need is @DavidKarlas' dotnet/templating#3078 fix.

Reply:
I needed to update this to get EmptyWebTemplateTest.EmptyWebTemplateCSharp EmptyWebTemplateTest.EmptyWebTemplateFSharp passing locally and on helix, so there's definitely room for improvement here. Not sure how common it is to update the templates to use very-newly-added APIs.

Reply:
My understanding of the fix to templating is that the regression broke the custom hive logic we were using to make our template tests find our updated templates, so that's why stefan had to update the sdk.  The templating workaround PR should also fix the tests I would assume